### PR TITLE
11  blog on rails

### PR DESCRIPTION
**Pull request by starspace1**
TheIronYard--Orlando/2015--SUMMER--ROR#85

### CODE REVIEW

File: first_blog/config/routes.rb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  get 'posts/:id/delete' => 'posts#delete', as: :delete_post
+
+  post 'posts/:post_id/comments' => 'comments#create', as: :post_comments
+
+  patch 'posts/:post_id/comments/:comment_id' => 'comments#update', as: :post_comment
+
+  get 'posts/:post_id/comments/:comment_id/edit' => 'comments#edit', as: :edit_comment
+
+  get 'posts/:post_id/comments/:comment_id/delete' => 'comments#delete', as: :delete_comment
```
Comment (bgates) [inline first_blog/config/routes.rb:23]: that shouldn't be a GET request, bc it's destructive (same w delete_post)

Comment (starspace1) [inline first_blog/config/routes.rb:23]: How does it work then?

File: first_blog/app/models/post.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+  has_many :comments
+
+  def preview
+
+    if(text.length < 100)
+      text
+    else
+      text[0..99]
+    end
+
```
Comment (bgates) [inline first_blog/app/models/post.rb:11]: Nice. There's also a method in ActionView which does something similar.

File: first_blog/app/controllers/posts_controller.rb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    @comment = Comment.new
+  end
+
+  def edit
+    @post = Post.find(params[:id])
+  end
+
+  def create
+    Post.create(params.require(:post).permit(:title, :author, :text))
+    redirect_to '/posts'
```
Comment (bgates) [inline first_blog/app/controllers/posts_controller.rb:22]: more idiomatic to redirect_to posts_path

**Pull request merged**